company-profiles/tuft-and-needle.md

```diff
@@ -0,0 +1,35 @@
+# Tuft & Needle
+
```

Comment:
I noticed that other companies included a link to their website below the company header. Should we do the same?